syntax = "proto2";
import "game_commands.proto";
message Command_Mulligan {
    extend GameCommand {
        optional Command_Mulligan ext = 1004;
    }
}


